Doc/whatsnew/3.14.rst

```diff
@@ -443,6 +454,8 @@ Python without deferred evaluation of annotations, reaches its end of life in 20
 In Python 3.14, the behavior of code using ``from __future__ import annotations``
 is unchanged.
 
+(Contributed by Jelle Zijlstra in :gh:`119180`; :pep:`649` was written by Larry Hastings.)
```

Comment:
Also add credit for  PEP 749?
```suggestion
(Contributed by Jelle Zijlstra in :gh:`119180`; :pep:`649` was written by Larry Hastings.)
```

Reply:
I think your suggestion is the same text as before? I don't think PEP 749 needs to be mentioned here again as my name is already in this line.

Reply:
Sorry! I didn't mean to do a suggestion here, was just meant to be the question.

Reply:
Thanks for the review!